libs/website-e2e/features/lighthouse/lighthouse.ts

```
@@ -0,0 +1,66 @@
+import type { Browser } from 'playwright'
+import { chromium, test } from '@playwright/test'
+import getPort from 'get-port'
+import { playAudit } from 'playwright-lighthouse'
+
+const lighthouseTest = test.extend<{ Page }, { port: number, browser: Browser }>({
+  port: [
+    // eslint-disable-next-line no-empty-pattern
+    async ({ }, use) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)